src/tools/pub/dependencies.md

```diff
@@ -16,13 +16,14 @@ At the end is a list of
 
 ## Overview
 
-For each dependency, you specify the *name* of the package you depend on
-and the *range of versions* of that package that you allow.
-You can also specify the
-[*source*](/tools/pub/glossary#source),
-which tells pub how to locate the package.
+For each dependency, you specify the _name_ of the package you depend on
+and the _range of versions_ of that package that you allow.
+You can also specify the [_source_][].
+The source tells pub how to locate the package.
 
-Here's an example of specifying a dependency:
+[_source_]: {{site.url}}/tools/pub/glossary#source
+
+To specify a dependency:
```

Comment:
The old text does make clear that this is an example.

To me it wrong to claim that this how an example